app/forms/gpo_verify_form.rb

```diff
@@ -17,15 +17,16 @@ def initialize(user:, pii:, otp: nil)
 
   def submit
     result = valid?
-    threatmetrix_check_failed = fraud_review_checker.fraud_check_failed?
+    threatmetrix_check_failed = pending_profile&.fraud_pending_reason.present?
```

Comment:
Wondering if it would make sense to add a method to `fraud_review_checker` for `fraud_pending_reason`. On the other hand, the code below reads clearly with this variable.

Reply:
Should we rename `threatmetrix_check_failed` to `fraud_check_failed`? To keep it vendor agnostic.

Reply:
Yeah, we can do that here